doc/develop/getting_started/index.rst

```diff
@@ -119,7 +119,7 @@ The current minimum required version for the main dependencies are:
 
          .. code-block:: bash
 
-            brew install cmake ninja gperf python3 ccache qemu dtc wget libmagic
+            brew install cmake ninja gperf python@3.8 ccache qemu dtc wget
```

Comment:
This is, in my opinion, wrong. Requiring every Zephyr user to install an outdated version of Python just because a tiny fraction of them will want to debug with Python support is just not right. Instead I think we should document this in the Zephyr SDK page in the doc.

Reply:
that's indeed a good point, I actually use Python 3.10 but I also have 3.8 installed for GDB and everything works fine (using FC36). And there's the `no-py` version, right? Maybe add a note about this requirement if using gdb.

Reply:
This is a good point. The initial goal here was to ensure that a user getting started with Zephyr would be able to debug using GDB, as GDB in our toolchain required python support. However, I am looking at the latest 0.15.0 SDK build, and it looks like GDB now defaults to not using python, and only requires it when running the `gdb-py` version of the executable. (@stephanosio could you confirm this?) If that is the case, I am completely fine dropping this, I just want to ensure the first thing users encounter when running `west debug` isn't a python dependency error